js/interface.js

```diff
@@ -307,32 +311,115 @@
     return Fliplet.API.request({
       url: [
         'v1/widgets?include_instances=true&tags=type:menu',
+        '&include_all_versions=true',
         '&appId=' + Fliplet.Env.get('appId'),
-        '&organizationId=' + (Fliplet.Env.get('organizationId') || '')
+        '&organizationId=' + (organizationId || '')
       ].join('')
     }).then(function(response) {
       return Promise.resolve(response.widgets);
     });
   }
 
+  /**
+   * Function to filter organizationMenus by version.
+   * @param {Array} data - array of organizationMenus
+   * @param {Object} latestVersionMenu - current active menu
+   * @returns {Object} latestVersionMenu - object which contains the latest version of current menu package type
+   */
+
+  function sortByVersion(data, latestVersionMenu) {
+    var previousVersion;
+
+    data.forEach(function(menu) {
+      // Compare each record with the previous one by version to find the newest version.
+      if (previousVersion && previousVersion.version !== menu.version) {
+        latestVersionMenu = (previousVersion.version > menu.version) ? previousVersion : menu;
+
+        return;
+      }
+
+      previousVersion = menu;
+
+      // Always write the system menu first, then if iteration finds the newest version, latestVersionMenu will be overwritten.
+      if (!latestVersionMenu) {
+        latestVersionMenu = menu;
+      }
+    });
+
+    return latestVersionMenu;
+  }
+
+  /**
+   * Function which filtering current array of menus by version and created a new one.
```

Comment:
"Function which filtering" is not good English and is also not necessary. The JSDoc is for a function so there's no need to mention "function which...". You can start with saying what it does.

```suggestion
   * Creates a list of menu widgets to be displayed based on versions and current usage
```